bundle/src/bootstraps/standalone.commercial.ts

```diff
@@ -212,48 +211,43 @@ const bootCommercial = async (): Promise<void> => {
 	}
 };
 
+const bootCommercialWhenReady = () => {
+	if (window.guardian.mustardCut || window.guardian.polyfilled) {
+		void bootCommercial();
+	} else {
+		window.guardian.queue.push(bootCommercial);
+	}
+};
+
 /**
  * Choose whether to launch Googletag or Opt Out tag (ootag) based on consent state
  */
 const chooseAdvertisingTag = async () => {
 	const consentState = await onConsent();
-	// Only load the Opt Out tag in TCF regions when it is switched on and there is no consent for Googletag
-	if (consentState.tcfv2 && !getConsentFor('googletag', consentState)) {
-		// Don't load OptOut (for now) if loading Elements Manager
-		if (isInVariantSynchronous(elementsManager, 'variant')) {
-			return;
-		}
-
+	// Only load the Opt Out tag if:
+	// - in TCF region
+	// - no consent for Googletag
+	// - the user is not a subscriber
+	if (
+		consentState.tcfv2 &&
+		!getConsentFor('googletag', consentState) &&
+		!isDigitalSubscriber()
```

Comment:
I don't think `isDigitalSubscriber` should be used directly, I think it's just a part of why someone might be ad free.

Reply:
Actually nevermind I think I'm thinking of one of the other identity cookies